docs/blog/2016-9-28.md

```diff
@@ -0,0 +1,15 @@
+# New Release
+We continue our montly release cycle with our [September release](https://github.com/vmware/docker-volume-vsphere/releases/tag/0.7)
+
+Thanks to [Bruno Moura's](https://github.com/brunotm) excellent [work](https://github.com/vmware/docker-volume-vsphere/pull/597), Docker volume plugin for vSphere now supports selection of file system format when creating a new docker volume.
+
+Ability to set a volume [read-only](https://github.com/vmware/docker-volume-vsphere/issues/274) is now supported by the plugin. The [admin cli](https://vmware.github.io/docker-volume-vsphere/user-guide/admin-cli/#set) allows the admin to set a volume to be read only.
```

Comment:
I agree with the both of you, but I think Govindan's text should go in Readme or doc in the same PR